app/locales/en/translations.js

```diff
@@ -474,6 +477,24 @@ export default {
                 base: 'This {{documentType.singular}}',
             },
         },
+        'author-assertions': {
+            no: 'No',
+            yes: 'Yes',
+            conflict_of_interest: {
+                title: 'Conflict of Interest',
+                no: 'Author asserted no Conflict of Interest',
```

Comment:
maybe check if this is the text we want (it matches the mockup but is inconsistent with the other text)